UPGRADE-4.0.md

```diff
@@ -16,6 +16,25 @@ Form
 
  * Support for data objects that implements both `Traversable` and
    `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed.
+   
+ * Using callable strings as choice options in ChoiceType has been removed in favor of passing PropertyPath instances.
+    
```

Comment:
I would replace "has been removed" by "is not supported anymore".

But please wait for a symfony decider to tell you if it's ok. Thanks :)